### PR TITLE
[FEAT] Display a BPMN diagram

### LINKED ISSUE #8: [FEAT] Display a BPMN diagram
If we decide to use `htmlwidgets`, see http://www.htmlwidgets.org/develop_intro.html#scaffolding for starting the integration

### PR DESCRIPTION
**Pull request by csouchet**
Closes #8

### CODE REVIEW

File: R/bpmnVisualization.R [hunk 1]
```diff
@@ -59,13 +59,13 @@ display <- function(bpmnXML, width = NULL, height = NULL, elementId = NULL) {
 #' @name bpmnVisualization-shiny
 #'
 #' @export
-bpmnVisualizationOutput <- function(outputId, width = '100%', height = '400px') {
+bpmn_visualization_output <- function(outputId, width = '100%', height = '400px') {
```
Comment (tbouffard) [inline R/bpmnVisualization.R:62]: ⚠️ htmlWidgets expects a function named `<package_name>Output` for the R-binding: http://www.htmlwidgets.org/develop_intro.html#r-binding
This doesn't follow our function naming convention

File: R/bpmnVisualization.R [hunk 2]
```diff
@@ -59,13 +59,13 @@ display <- function(bpmnXML, width = NULL, height = NULL, elementId = NULL) {
...
   htmlwidgets::shinyWidgetOutput(outputId, 'bpmnVisualization', width, height, package = 'bpmnVisualization')
 }
 
 #' @rdname bpmnVisualization-shiny
 #' @export
-renderBpmnVisualization <- function(expr, env = parent.frame(), quoted = FALSE) {
+render_bpmn_visualization <- function(expr, env = parent.frame(), quoted = FALSE) {
```
Comment (tbouffard) [inline R/bpmnVisualization.R:68]: ⚠️ htmlWidgets expects a function named `render<upper_case_package_name>` for the R-binding: http://www.htmlwidgets.org/develop_intro.html#r-binding

File: R/bpmnVisualization.R (hunk 1 above)
Comment (csouchet) [reply R/bpmnVisualization.R:62]: Yes, we can't rename this function.

Comment (tbouffard) [reply R/bpmnVisualization.R]: done in f9fe264

File: R/bpmnVisualization.R (hunk 2 above)
Comment (tbouffard) [reply R/bpmnVisualization.R]: done in f9fe264

**Review by tbouffard — APPROVED** [review]

**Pull request merged**